src/ui/events.js

```diff
@@ -266,47 +292,122 @@ export type MapEvent =
     /**
      * Fired when a pointing device (usually a mouse) is pressed within the map.
      *
+     * **Note:** This event is compatible with the optional `layerId` parameter.
```

Comment:
Please confirm that these new notes in this file are consistent with other notes format.
It's sometimes only described as such 
https://github.com/mapbox/mapbox-gl-js/blob/8c7c88332ecf515555a308d38117440fa22be126/src/ui/camera.js#L874

Reply:
@karimnaaji I'm not sure what you mean. I don't think there is any established convention in these docs for how to format asides like this. If there is a convention, we could either change these new notes to match or go through and make everything consistent in some future work. My preference would be to tackle it later since my impression is that the approach thus far has been pretty ad hoc.

Reply:
For this I meant, within `**` vs not.